Dizzy/Screens/PlaceStory/Comments/CommentsView.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  CommentView.swift
+//  Dizzy
+//
+//  Created by Tal Ben Asuli on 07/05/2019.
+//  Copyright © 2019 Dizzy. All rights reserved.
+//
+
+import UIKit
+
+protocol CommentsViewDelegate: class {
```

Comment:
i think comments manager should be combine with this class and do everything here